core/src/checks/self_checks.c

```diff
@@ -54,6 +54,7 @@ static void register_all_self_checks(void) {
   REGISTER_SELF_CHECK(verify_no_rollback);
   REGISTER_SELF_CHECK(verify_no_rollback_write_to_buf);
   REGISTER_SELF_CHECK(verify_conv_btc_to_satoshi);
+  REGISTER_SELF_CHECK(verify_wycheproof);
```

Comment:
super nit: prefer more consistency between declaration order in checks.h and execution order in self_checks.c